scripts/tags/exturl.js

```diff
@@ -4,8 +4,11 @@
 
 'use strict';
 
-var util = require('hexo-util');
+/*jshint camelcase: false */
+var util = require(hexo.base_dir + 'node_modules/hexo/node_modules/hexo-util');
```

Comment:
No. `hexo-util` module would not exist in the `hexo/node_modules/` directory if using NPM whose version is above 3 to install hexo.

Please see the following testing result:

1. Install with NPM 2 (The version that you used)
![npm2](https://cloud.githubusercontent.com/assets/32269/23962783/310ccbc4-09ea-11e7-9659-ea8534694c1b.png)

1. Install with NPM 3
![npm3](https://cloud.githubusercontent.com/assets/32269/23962807/403f1930-09ea-11e7-87f4-c4d33a4a1322.png)

You could see that, `hexo-util` module does not exist in the `hexo/node_modules` directory when installed with NPM 3.  So the solution for this issue is that we could just use `require('hexo-util')` in the tag scripts and inform users to upgrade their NPM or install missing modules (`hexo-util` in this case) if errors occur to simplify the process.

--------

For your information, the following is what I did in this testing:

```
mkdir test-with-npm2 && cd test-with-npm2
nvm use v0.10.48
npm init
npm install --save-dev hexo
./info.sh

mkdir test-with-npm3 && cd test-with-npm3
nvm use v7.6.0
npm init
npm install --save-dev hexo
./info.sh
```

The content of `info.sh` file is:

```sh
#!/bin/bash

printf 'Node.js version: %s\n' $(node -v)
printf 'NPM version: %s\n' $(npm -v)
printf 'hexo-util module in hexo sub-direcotry: \n'
printf '  ' && ls -ld node_modules/hexo/node_modules/hexo-util
printf 'hexo-util nodele in root sub-directory: \n'
printf '  ' && ls -ld node_modules/hexo-util
```